### PR TITLE
fix(console): angular 13, grpc web 1.3.0, dependencies

### PR DESCRIPTION
**Pull request by peintnermax**

### CODE REVIEW

File: build/zitadel/Dockerfile [hunk 1]
```diff
@@ -20,7 +20,7 @@ ARG PROTOC_VERSION=3.18.0
 ARG PROTOC_ZIP=protoc-${PROTOC_VERSION}-linux-${PROTOC_ARCH}.zip
 ARG GATEWAY_VERSION=2.6.0
 ARG VALIDATOR_VERSION=0.6.2
-ARG GRPC_WEB_VERSION=1.2.1
+ARG GRPC_WEB_VERSION=1.3.0
```
Comment (livio-a) [inline build/zitadel/Dockerfile:23]: @adlerhurst protoc-gen-grpc-web is only required for the client side, right?

Comment (adlerhurst) [reply build/zitadel/Dockerfile:23]: I would change the version in build/console/Dockerfile.

Comment (livio-a) [reply build/zitadel/Dockerfile:23]: it has been changed there as well
so imo we can remove all of it in this file

Comment (adlerhurst) [reply build/zitadel/Dockerfile:23]: Yes

**Review by livio-a — APPROVED** [review]

**Pull request merged**